spec/support/capybara.rb

```diff
@@ -6,18 +6,19 @@
 require 'action_dispatch'
 
 RSpec.shared_context 'with default_url_options and host name set to Capybara test server' do
-  around do |example|
-    original_host = default_url_options[:host]
-    original_port = default_url_options[:port]
-    original_host_setting = Setting.host_name
+  before do
+    @original_host = default_url_options[:host]
+    @original_port = default_url_options[:port]
+    @original_host_setting = Setting.host_name
+    Setting.host_name = "#{Capybara.server_host}:#{Capybara.server_port}"
     default_url_options[:host] = Capybara.server_host
     default_url_options[:port] = Capybara.server_port
-    Setting.host_name = "#{Capybara.server_host}:#{Capybara.server_port}"
-    example.run
-  ensure
-    default_url_options[:host] = original_host
-    default_url_options[:port] = original_port
-    Setting.host_name = original_host_setting
+  end
+
+  after do
+    default_url_options[:host] = @original_host # rubocop:disable RSpec/InstanceVariable
+    default_url_options[:port] = @original_port # rubocop:disable RSpec/InstanceVariable
+    Setting.host_name = @original_host_setting # rubocop:disable RSpec/InstanceVariable
```

Comment:
This is the fix Oliver did, but the first version only. Then it was transformed to a `around` instead of `before`/`after` so that we could get rid of the instance variables (and make rubocop happier to avoid the `rubocop:disable RSpec/InstanceVariable`).

Can you revert to the original code present on `dev`?